opm/parser/eclipse/EclipseState/Grid/Box.hpp

```diff
@@ -44,6 +47,11 @@ namespace Opm {
 
         Box(const EclipseGrid& grid);
         Box(const EclipseGrid& grid , int i1 , int i2 , int j1 , int j2 , int k1 , int k2);
+        Box(const EclipseGrid& grid, const DeckKeyword& keyword);
+        Box(const EclipseGrid& grid_arg, const DeckRecord& deckRecord);
```

Comment:
Do we have, or plan to have, client code that benefits from having both of these constructors?  Given that the `DeckKeyword` version just forwards `keyword.getRecord(0)` directly to the `DeckRecord` version I don't quite see why we the need for the former, especially since we could add a free function along the lines of
```C++
Box makeBox(const EclipseGrid& grid, const DeckKeyword& boxKeyword)
{
    if (boxKeyword.name() != "BOX") {
        throw std::invalid_argument {
            "Expected 'BOX' keyword, but got '" + boxKeyword.name() + '\''
        };
    }

    return { grid, boxKeyword.getRecord(0) };
}
```